app/endpoints/elocaps.py

```diff
@@ -0,0 +1,158 @@
+from datetime import date
+
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.cruds import cruds_elocaps
+from app.dependencies import get_db, is_user_a_member
+from app.models import models_core, models_elocaps
+from app.schemas import schemas_elocaps
+from app.utils.types.elocaps_types import CapsMode
+from app.utils.types.tags import Tags
+
+router = APIRouter()
+
+
+@router.post(
+    "/elocaps/games",
+    status_code=201,
+    response_model=schemas_elocaps.Game,
+    tags=[Tags.elocaps],
+)
+async def register_game(
+    game_params: schemas_elocaps.GameCreateRequest,
+    db: AsyncSession = Depends(get_db),
+    user: models_core.CoreUser = Depends(is_user_a_member),
+):
+    if not any(user.id == i.user_id for i in game_params.players):
```

Comment:
Not sure about this suggestion. What do you think?

```suggestion
    if user.id not in [player.user_id for player in game_params.players):
```

Reply:
`all()` is maybe a bit more understandable